lib/Slackbot_worker.js

```diff
@@ -1,17 +1,21 @@
 var Ws = require('ws');
+var clone = require('lodash.clone');
 var request = require('request');
 var slackWebApi = require(__dirname + '/Slack_web_api.js');
 var HttpsProxyAgent = require('https-proxy-agent');
 
 
 module.exports = function(botkit, config) {
+    config = config || {};
     var bot = {
         botkit: botkit,
-        config: config || {},
+        config: clone(config),
         utterances: botkit.utterances,
-        api: slackWebApi(botkit, config || {})
+        api: slackWebApi(botkit, clone(config))
     };
 
+    var botApi = slackWebApi(botkit, bot.config);
```

Comment:
The bot API needs the bot token and the web api needs the `access_token` with extra scopes. The `bot.config` object has the bot token.